pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+name = "requests-unixsocket"
+version = "0.3.0"
+description = "Use requests to talk HTTP via a UNIX domain socket"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
A lot of people have already dropped 3.8 because it's EOL: https://endoflife.date/python